lib/sanbase/clickhouse/daily_active_addresses.ex

```diff
@@ -28,6 +28,14 @@ defmodule Sanbase.Clickhouse.DailyActiveAddresses do
   end
 
   def average_active_addresses(contract, from, to, interval) do
-    Erc20.average_active_addresses(contract, from, to, interval)
+    Erc20.average_active_addresses_with_deposits(contract, from, to, interval)
```

Comment:
I suspect this needs to be 

```suggestion
    Erc20.average_active_addresses(contract, from, to, interval)
```